src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+/*
+ * Tagging object class, to record list of commands for tags used in session
```

Comment:
`Tagging object class,`? This phrase doesn't make sense.
The purpose of the class is to represent a Tagging, not a list of Taggings.